resources/terminology.md

```diff
@@ -0,0 +1,71 @@
+    - url: /resources/development-guidelines
+      title: Open Source Development Guidelines
+    - url: /resources/terminology
+      title: Terminology
+---
+
+{% include alert-development-section.md %}
+
+{:.table .table-striped}
+| Term                            | Description                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                 |
+| :------------------------------ | :-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| **Access Token**                | The OAauth 2 Access token needed to access Swedbank Pay eCom APIs. Tokens are generated in Swedbank Pay Admin. Learn how to getting started in the Admin Manual.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                            |
+| **Account Receivable Consumer** | The [`AccountReceivableConsumer`][invoice-url] API is the fundament for Swedbank Pay [Invoice Payments][invoice-url] service. It is a service where Swedbank Pay helps produce and distribute invoices to consumers/end-users.                                                                                                                                                                                                                                                                                                                                                                                                                                              |
```

Comment:
It's not obvious from this description how `AccountReceivableConsumer` is different to `FinancingConsumer`. Both of these concepts should be more thorougly explained in Invoice Payments Other Features. Can you add a task for it?

Reply:
Yes, creating a task for 1) and 2) and invoice payments other features 3)